web/site/app.vue

```diff
@@ -17,15 +17,21 @@ useHead({
 
 const appVersion = await getAppMetaInfo() // load ui and api version on app mount
 
-// query help markdown and globally provide to use in either pages/help.vue or <SbcHelpModal />
-// if queried directly from modal, it wont be prerendered because modal is hidden initially
-const { data: helpDocs } = await useAsyncData('help-docs-query', () => {
+// Query help markdown and globally provide it for use in either pages/help.vue or <SbcHelpModal />
+const { data: helpDocs, refresh: refreshHelpDocs } = await useAsyncData('help-docs-query', () => {
```

Comment:
Can you not use the watch option provided by useAsyncData?

https://nuxt.com/docs/api/composables/use-async-data#watch-params